hack/render-manifests.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+	"text/template"
+
+	"github.com/pkg/errors"
+)
+
+func exitWithError(err error, cause string, args ...interface{}) {
+	fmt.Fprintf(os.Stderr, "render-manifests.go: error: %v\n", errors.Wrapf(err, cause, args))
+	os.Exit(1)
+}
+
+func main() {
+	type Inventory struct {
+		Namespace           string
+		NMStateHandlerImage string
+		ImagePullPolicy     string
+	}
+
+	inventory := Inventory{
+		Namespace:           os.Args[1],
+		NMStateHandlerImage: os.Args[2],
+		ImagePullPolicy:     os.Args[3],
+	}
+	inputDir := os.Args[4]
+	outputDir := os.Args[5]
+
+	// Clean up output dir so we don't have old files.
+	err := os.RemoveAll(outputDir)
+	if err != nil {
+		exitWithError(err, "failed cleaning up output dir %s", outputDir)
+	}
+
+	tmpl, err := template.ParseGlob(path.Join(inputDir, "*.yaml"))
+	if err != nil {
+		exitWithError(err, "failed parsing top dir manifests at %s", inputDir)
+	}
+
+	tmpl, err = tmpl.ParseGlob(path.Join(inputDir, "*/*.yaml"))
+	if err != nil {
+		exitWithError(err, "failed parsing sub dir manifests at %s", inputDir)
+	}
+
+	for _, t := range tmpl.Templates() {
+		manifestDir := outputDir
+
+		// We don't have the full path from ParseGlon so we just guess here
+		// this is not going to change too much
+		if strings.Contains(t.Name(), "scc.yaml") {
+			manifestDir = path.Join(manifestDir, "openshift")
+		} else if strings.Contains(t.Name(), "_crd.yaml") || strings.Contains(t.Name(), "_cr.yaml") {
+			manifestDir = path.Join(manifestDir, "crds")
```

Comment:
Do we need subdirectories at all? Can't we just keep it in a single one?

Reply:
Just one to make it similar ot template dir so we don't have to change much stuff at release scripts.

Reply:
Have refactor and now is everything at the same dir.